tests/integration/s2n_client_endpoint_handshake_test.py

```diff
@@ -81,12 +81,20 @@ def try_client_handshake(endpoint, arguments, expected_cipher):
 
     return 0
 
-def well_known_endpoints_test(use_corked_io):
+def well_known_endpoints_test(use_corked_io, tls13_enabled):
+
     arguments = []
     msg = "\n\tTesting s2n Client with Well Known Endpoints:"
+    if tls13_enabled:
+        arguments += ["--tls13", "--ciphers", "default_tls13"]
+        msg = msg[:-1] + " using TLS 1.3:"
     if use_corked_io:
-        msg = "\n\tTesting s2n Client with Well Known Endpoints using Corked IO:"
         arguments += ["-C"]
+        if tls13_enabled:
+            msg = msg[:-1] + " and Corked IO:"
+        else:
+            msg = msg[:-1] + " using Corked IO:"
```

Comment:
I see you have taking account the grammar with the different permutations :) It might be harder tho to maintain if more options get added in future (which we could then make it easier to display the options passed in) but I think it's great for now.

Reply:
You could keep a list of options being used, and `','.join(opt_list)` that list to the end of the message. This would be an easier way to maintain the grammar.

Reply:
Good advice, I wrote something to this effect to provide for future options. It also uses an oxford comma for the list, which I know can be controversial but I think it makes it more clear here that these are all separate options.